images/istio/image.yaml

```diff
@@ -0,0 +1,3 @@
+versions:
+  - apko:
+      config: proxy/configs/latest.apko.yaml
```

Comment:
nit: configs/proxy/latest.apko.yaml, but this isn't semantic anymore :)

Reply:
Done